### PR TITLE
Issue 3449

### PR DESCRIPTION
**Pull request by ltkum**
in issue #3449 , we have a problem to display kmls in the admin page for them. 

After investigation, it seems the root of the issue is that we wrote the wrong bucket names in dynamodb as we save kmls.

### CODE REVIEW

File: chsdi/lib/files_handler.py [hunk 1]
```diff
@@ -121,8 +123,7 @@ class FilesHandler(object):
     def __init__(self, request):
         self.request = request
         # Set up AWS DynamoDB and S3 handlers
-        self.dynamodb_fileshandler = DynamoDBFilesHandler(
-            self.dynamodb_table_name, self.bucket_key_name, self.region)
+        self.dynamodb_fileshandler = DynamoDBFilesHandler(self.dynamodb_table_name, self.bucket_name, self.region)
```
Comment (procrastinatio) [inline chsdi/lib/files_handler.py:126]: Do we use this `bucket_key_name` anywhere?

File: chsdi/views/admin.py [hunk 2]
```diff
@@ -36,17 +42,27 @@ def kml_load(api_url='//api3.geo.admin.ch', bucket_name=None, table_name=None, r
...
+    logging.debug("----------------------- ! ! ! ---------------------")
     for f in results['Items']:
         try:
             resp = requests.head("http:" + api_url + "/files/" + f['fileId'], headers={'User-Agent': 'mf-geoadmin/python'})
             if int(resp.status_code) == 200:
                 fileids.append((f['fileId'], f['adminId'], f['timestamp']))
-        except RequestException:
-            pass
+        except RequestException as e:
+            logging.error(e)
```
Comment (procrastinatio) [inline chsdi/views/admin.py:65]: Nice

File: chsdi/lib/files_handler.py [hunk 3]
```diff
@@ -39,7 +39,10 @@ def get_item(self, admin_id):
         item = None
         try:
             item = self.table.get_item(Key={'adminId': str(admin_id)}).get('Item', None)
-        except Exception:
+        except Exception as e:
+            log.error("----------------------------------------")
```
Comment (procrastinatio) [inline chsdi/lib/files_handler.py:43]: Cleanup?

File: chsdi/views/admin.py [hunk 4]
```diff
@@ -18,6 +19,11 @@ def admin_kml(context, request):
     table_name = settings.get('geoadmin_file_storage_table_name')
     table_region = settings.get('geoadmin_file_storage_table_region')
     api_url = settings.get('api_url')
+    logging.debug("-----------------------------------------------------------------------")
+    logging.debug(bucket_name)
```
Comment (procrastinatio) [inline chsdi/views/admin.py:23]: Not sure if we leave this. Maybe add the name of the variable whose value is printed

**Comment by procrastinatio** [comment]
Rebase you branch on `master`. The tests will pass (hopefully)

File: chsdi/lib/files_handler.py (hunk 1 above)
Comment (ltkum) [reply chsdi/lib/files_handler.py:126]: From what I've seen, not anymore.

File: chsdi/lib/files_handler.py (hunk 3 above)
Comment (ltkum) [reply chsdi/lib/files_handler.py:43]: It's still a work in progress (I should provide a test link), I'm keeping some logs here and there. But I'll cleanup most of the logs before going live

File: chsdi/views/admin.py (hunk 4 above)
Comment (ltkum) [reply chsdi/views/admin.py:23]: In the long run, no real need to keep most of the debug logs in my opinion.

**Comment by procrastinatio** [comment]
Looks OK. Ich teste noch.

**Comment by procrastinatio** [comment]
Oops, wrong language.

**Pull request merged**